pkg/project/auth/reviewer.go

```diff
@@ -0,0 +1,55 @@
+package auth
+
+import (
+	"github.com/openshift/origin/pkg/client"
+)
+
+// Review is a list of users and groups that can access a resource
+type Review interface {
+	Users() []string
+	Groups() []string
+}
+
+// Reviewer performs access reviews for a project by name
+type Reviewer interface {
+	Review(name string) (Review, error)
+}
+
+// review implements the Review interface
+type review struct {
+	users  []string
+	groups []string
+}
+
+// Users returns the users that can access a resource
+func (r *review) Users() []string {
+	return r.users
+}
+
+// Groups returns the groups that can access a resource
+func (r *review) Groups() []string {
+	return r.groups
+}
+
+// reviewer performs access reviews for a project by name
+type reviewer struct {
+	client client.Interface
+}
+
+// NewReviewer knows how to make access control reviews for a resource by name
+func NewReviewer(client client.Interface) Reviewer {
+	return &reviewer{
+		client: client,
+	}
+}
+
+// Review performs a resource access review for the given resource by name
+func (r *reviewer) Review(name string) (Review, error) {
+	review := &review{
+		users:  []string{},
+		groups: []string{},
+	}
+	// TODO: make a client request to look-up the project from the client
+	// TODO: chat with deads2k on if we have a project and a namespace what to do on acl sync
```

Comment:
You're looking to make a call to here: https://github.com/openshift/origin/blob/master/pkg/authorization/authorizer/authorizer.go#L24